project.clj

```diff
@@ -9,18 +9,19 @@
                  ^:source-dep [cider/orchard "0.4.0"]
                  ^:source-dep [thunknyc/profile "0.5.2"]
                  ^:source-dep [mvxcvi/puget "1.1.0"]
-                 ^:source-dep [fipp "0.6.15"]
+                 ^:source-dep [fipp "0.6.15"]; can be removed in unresolved-tree mode
                  ^:source-dep [compliment "0.3.8"]
                  ^:source-dep [cljs-tooling "0.3.1"]
                  ^:source-dep [cljfmt "0.6.4" :exclusions [org.clojure/clojurescript]]
-                 ;; Not used directly in cider-nrepl, but needed because of tools.namespace
-                 ;; and the way MrAnderson processes dependencies
-                 ;; See https://github.com/clojure-emacs/cider/issues/2176 for details
-                 ^:source-dep [org.clojure/java.classpath "0.3.0"]
                  ^:source-dep [org.clojure/tools.namespace "0.3.0-alpha4"]
                  ^:source-dep [org.clojure/tools.trace "0.7.10"]
                  ^:source-dep [org.clojure/tools.reader "1.2.2"]]
-  :plugins [[thomasa/mranderson "0.4.9"]]
+  :plugins [[thomasa/mranderson "0.5.0"]]
+  :mranderson {:project-prefix "cider.inlined-deps"
+               :overrides       {[mvxcvi/puget fipp] [fipp "0.6.15"]}; only takes effect in unresolved-tree mode
+               :expositions     [[mvxcvi/puget fipp]]; only takes effect unresolved-tree mode
```

Comment:
Btw, shouldn't this be named `exposes` to be consistent with `overrides`. I guess it's consistent in a way even now, but I've never heard expositions used in this sense. :-)

Reply:
as much as I googled it seemed that the noun form is `exposition`. eg something being exposed is an exposition (or an exposure but that has negative connotations).

Reply:
Well, I can live with the name. :-) My actually confusion was from the fact that `overrides` can be interpreted both as a verb and a noun and first I thought it was a verb. No a big deal.